sdk/python/tests/components/test_components.py

```diff
@@ -62,26 +62,25 @@ def test_load_component_from_zipped_yaml_file(self):
         component_path = _test_data_dir.joinpath('python_add.component.zip')
         self._test_load_component_from_file(str(component_path))
 
-    @unittest.skip
-    @unittest.expectedFailure #The repo is non-public and will change soon. TODO: Update the URL and enable the test once we move to a public repo
+    #The repo is non-public and will change soon. TODO: Update the URL and enable the test once we move to a public repo
```

Comment:
Not sure what this comment mean. Is it a stale one?

Reply:
It's stale. I'll remove it.

Reply:
I've removed that line.